packages/web-components/src/components/leaving-ibm/leaving-ibm.scss

```diff
@@ -34,6 +34,6 @@
 :host(#{$dds-prefix}-leaving-ibm-modal-supplemental) {
   display: block;
   margin-top: $spacing-07;
-  color: $text-02;
   @include carbon--type-style('label-01');
+  @include carbon--type-style(body-long-01, true);
```

Comment:
Is `label-01` still needed?

Reply:
I got rid of `label-01` for web-components but still needed for the react-wrapper fix